lib/screens/home_screen.dart

```diff
@@ -0,0 +1,151 @@
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.only(top: 50, left: 20, right: 20),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Icon(
+                    Icons.search,
+                    color: Colors.grey,
+                    size: 34.0,
+                  ),
+                  Column(
+                    children: [
+                      Text(
+                        "Make Home",
+                        style: TextStyle(
+                          color: Color(0xff606060),
+                          fontFamily: 'Gelasio',
+                          fontSize: 20.0,
+                          fontWeight: FontWeight.w600,
+                          height: 1.25,
+                          letterSpacing: 1.0,
+                        ),
+                      ),
+                      Text(
+                        "Beautiful",
+                        style: TextStyle(
+                          color: Color(0xff303030),
+                          fontFamily: 'Gelasio',
+                          fontSize: 30.0,
+                          fontWeight: FontWeight.w700,
+                          height: 1.26,
+                          letterSpacing: 1.0,
+                        ),
+                      )
+                    ],
+                  ),
+                  Icon(
+                    Icons.shopping_cart,
+                    color: Colors.grey,
+                    size: 34.0,
+                  ),
+                ],
+              ),
+            ),
+            const SizedBox(height: 30),
+            SizedBox(
+              height: 100,
+              width: size.width,
+              child: ListView.separated(
+                scrollDirection: Axis.horizontal,
+                itemBuilder: (context, index) {
+                  return Column(
+                    children: [
+                      Container(
+                        width: 50.0,
+                        height: 50,
+                        decoration: BoxDecoration(
+                          color: const Color(0xfff5f5f5),
+                          borderRadius: BorderRadius.circular(12),
+                          border: Border.all(color: Colors.white),
+                        ),
+                        padding: const EdgeInsets.all(8.0),
+                        child: Center(child: Image.asset(items[index].image)),
+                      ),
+                      const SizedBox(height: 8.0),
+                      Text(
+                        items[index].name,
+                        style: const TextStyle(
+                          color: Colors.grey,
+                          fontWeight: FontWeight.bold,
+                        ),
```

Comment:
Hardcoding colors directly to widgets is not recommended, so we should be referring them from theme

like

Theme.of(context).primaryColor